subprojects/plugin-development/src/main/java/org/gradle/plugin/devel/tasks/GeneratePluginDescriptors.java

```diff
@@ -36,6 +37,7 @@
 /**
  * Generates plugin descriptors from plugin declarations.
  */
+@DisableCachingByDefault(because = "Is not CPU intensive")
```

Comment:
Not CPU intensive does not seem like a good explanation. It's more about the task doesn't take a long time to execute, no?